okta/provider.go

```diff
@@ -47,10 +50,13 @@ func Provider() terraform.ResourceProvider {
 			signOnPolicy:             resourceSignOnPolicy(),
 			signOnPolicyRule:         resourceSignOnPolicyRule(),
 			passwordPolicyRule:       resourcePasswordPolicyRule(),
+			mfaPolicy:                resourceMfaPolicy(),
```

Comment:
Do these camelCase resources only serve as sub-resources of the actual resources used by users, or does terraform's provider API know to make them snake_case?

Reply:
They just get dumped into this map and used by consumers via terraform. I use constants internally since the resource name is used elsewhere. Especially the tests, they are referenced quite a few times.